### PR TITLE
Slack Integration: Update other-advanced-features.md

### PR DESCRIPTION
**Pull request by lykoh01**
Added the link to authorize slack workspace and remove the "Add to Slack" icon.

### CODE REVIEW

File: pages/docs/analysis/advanced/other-advanced-features.md [hunk 1]
```diff
@@ -77,11 +77,7 @@ The Mixpanel application for Slack will also automatically unfurl a preview of
 
 ### Enable the Integration
 
-To enable the integration, log in to both Mixpanel and Slack, then click "Add to Slack" below:
-
-![/add_to_slack.png](/add_to_slack.png)
-
-Then, click "Allow:"
```
Comment (mavlee) [inline pages/docs/analysis/advanced/other-advanced-features.md:84]: this line should be kept

**Comment by lykoh01** [comment]
https://github.com/mixpanel/docs/pull/167

**Comment by lykoh01** [comment]
close https://github.com/mixpanel/docs/pull/167 instead.

File: pages/docs/analysis/advanced/other-advanced-features.md (hunk 1 above)
Comment (ranic) [reply pages/docs/analysis/advanced/other-advanced-features.md:84]: Seems fine to me? 
https://docs-git-lykoh01-patch-2-mixpanel.vercel.app/docs/analysis/advanced/other-advanced-features#enable-the-integration

**Pull request merged**